src/logger/loggerApi.c

```diff
@@ -267,6 +268,16 @@ int api_getCapabilities(struct Serial *serial, const jsmntok_t *json)
         return API_SUCCESS_NO_RETURN;
 }
 
+static void imu_status(struct Serial *serial, const bool more)
```

Comment:
suggest naming 'send_imu_status' - consistent with other similar functions. Similar comment in other PR.